src/lib/InPort.coffee

```diff
@@ -8,35 +8,17 @@ IP = require './IP'
 platform = require './Platform'
 
 class InPort extends BasePort
-  constructor: (options, process) ->
-    @process = null
-
-    if not process and typeof options is 'function'
-      process = options
-      options = {}
-
-    options ?= {}
-
-    options.buffered ?= false
+  constructor: (options = {}) ->
     options.control ?= false
     options.scoped ?= true
     options.triggering ?= true
 
-    if not process and options and options.process
-      process = options.process
+    if options.process
+      platform.deprecated 'InPort process callback is deprecated. Please use Process API'
```

Comment:
This deletes options.process afterwards, leading me to think using it does not work at all? If so this is not a depreciation warning but an Error (functional is already removed, not scheduled to be removed)

Reply:
Yeah, those should probably be direct throws now